torchaudio/functional/functional.py

```diff
@@ -1631,3 +1632,38 @@ def rnnt_loss(
         return costs.sum()
 
     return costs
+
+
+def compute_power_spectral_density_matrix(
```

Comment:
the name seems a little verbose. would something like `psd` suffice? would also be consistent with the existing transform `PSD`

Reply:
This is to align the [get_power_spectral_density_matrix](https://github.com/espnet/espnet/blob/master/espnet2/enh/layers/beamformer.py#L25) method in ESPNet and [get_power_spectral_density_matrix](https://github.com/fgnt/pb_bss/blob/master/pb_bss/extraction/beamformer.py#L59) in pb_bss. `compute` sounds more reasonable than `get`.

Reply:
we already appear to have a convention of names of functionals being the snake-case of the names of the corresponding transforms, e.g. `torchaudio.functional.inverse_spectrogram` and `torchaudio.transforms.InverseSpectrogram`, and `torchaudio.transforms.PSD` already exists. why would we prefer diverging to align with espnet?

Reply:
`psd` is fine for me.
For the other methods like `compute_mvdr_weights_souden`, there may not be a corresponding module, as the new beamforming module in release 0.12 will use both `compute_mvdr_weights_xx` and `apply_beamforming` to obtain the enhanced spectrogram directly. Do those method names LGTY?

Reply:
with a couple of exceptions, it seems like, if no corresponding concise verb exists, functionals are named as nouns. if we want to follow that convention, then `mvdr_weights_xx` would be more appropriate. also, adding `compute_` seems a little redundant since it's implicit that every functional computes something